### PR TITLE
cloud: add docs for playground and cluster overview

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: add docs for playground

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by langyuezhang — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-glossary.md [hunk 1]
```diff
@@ -55,6 +55,10 @@ Organization members are users who are invited by the organization owner to join
 
 ## P
 
+### Playground
+
+Playground contains a pre-loaded dataset of GitHub events, which allows Developer Tier users to get started with TiDB Cloud by running queries instantly, without importing data or connecting to a client.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-glossary.md:60]:
```suggestion
Playground contains a pre-loaded dataset of GitHub events, which allows Developer Tier users to get started with TiDB Cloud by running queries instantly, without importing data or connecting to a client.

You can access Playground after creating a Developer Tier cluster. 
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 2]
```diff
@@ -41,6 +41,10 @@ To create a free Developer Tier cluster, take the following steps:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:41]: After clicking **Create**, *the **Security Settings** dialog box will not be displayed. Users have to open it manually. Please update the steps here accordingly.   And the https://docs.pingcap.com/tidbcloud/create-tidb-cluster doc needs to be updated as well.

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 3]
```diff
@@ -35,11 +35,16 @@ To create a free Developer Tier cluster, take the following steps:
...
-6. In the **Security Settings** dialog box, set the root password and allowed IP addresses to connect to your cluster, and then click **Apply**.
+6. During the creation process, perform security settings for your cluster:
 
-    Your TiDB Cloud cluster will be created in approximately 5 to 15 minutes.
+    1. Click **Security Settings** in the upper-right corner of the cluster area.
+    2. In the **Security Settings** dialog box, set the root password and allowed IP addresses to connect to your cluster, and then click **Apply**.
+
+> **Tip:**
+>
+> After your TiDB Cloud cluster is created, and before importing your data or connecting to a client, you can run queries instantly on TiDB Cloud. Click **Playground** to have a try. Playground contains a pre-loaded dataset of GitHub events.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:47]: "before importing your data or connecting to a client, you can run queries instantly on TiDB Cloud" looks confusing. How about updating it to something like the following:

before importing your data or connecting to a client, you can click **Playground** to run queries instantly on TiDB Cloud.

Comment (TomShawn) [reply tidb-cloud/tidb-cloud-quickstart.md:47]:
```suggestion
> After your TiDB Cloud cluster is created, and before importing your data or connecting to a client, you can click **Playground** to run queries instantly on TiDB Cloud. Playground contains a pre-loaded dataset of GitHub events.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**